### PR TITLE
The KDE was renamed to Plasma

### LINKED ISSUE #1288: s/KDE/Plasma
## Description

Hello,

I found out today that Neofetch reports Plasma as KDE. KDE is the community and Plasma is the DE and one of the product of the KDE community. Could this be fixed?

Thanks

#### Neofetch version

6.0

## Screenshot

Screenshot_20190719_234236

### PR DESCRIPTION
**Pull request by ognarb**
## Description

Replace KDE with Plasma.

## Issues

Fix  #1288

### CODE REVIEW

**Comment by konimex** [comment]
KDE 4 was not branded Plasma, wasn't it? I'd add a check if it's < KDE 5.

**Comment by ognarb** [comment]
@konimex I'm now comparing that KDE_SESSION_VERSION is greater equal 4.

**Comment by konimex** [comment]
Otherwise, LGTM.

File: neofetch [hunk 1]
```diff
@@ -1560,6 +1560,9 @@ get_de() {
         ;;
     esac
 
+    if [ "$KDE_SESSION_VERSION" -ge "4" ]; then
```
Comment (konimex) [inline neofetch]: Instead of an if block, use `(( KDE_SESSION_VERSION > 4 )) && [...]` as it is a bash syntax.

Comment (ognarb) [reply neofetch]: KDE was renamed to Plasma during the KDE 4 days. https://en.wikipedia.org/wiki/KDE_Plasma_4

**Comment by konimex** [comment]
Merged. Thanks for the PR.

**Comment by ognarb** [comment]
Thanks for the review :)

**Pull request merged**